developer-docs-site/docs/tutorials/first-transaction.md

```diff
@@ -9,12 +9,12 @@ import TabItem from '@theme/TabItem';
 
 # Your first transaction
 
-This tutorial details how to generate, submit, and verify transactions submitted to the Aptos Blockchain. The steps to doing so:
+This tutorial outlines how to generate, submit, and verify transactions submitted to the Aptos Blockchain. The steps for doing so are:
```

Comment:
oxford comma is still here :).  Let's just keep the oxford commas in all of the sentences.

Also, I'd simplify again to `The steps are`